posts/comparing-quarkus-to-spring-through-the-eyes-of-jhipster.md

```diff
@@ -0,0 +1,120 @@
+---
+authors:
+- Dennis Sharpe
+tags:
+- Quarkus
+- Spring
+- JHipster
+- Java
+date: 2021-01-29T15:52:00.000Z
+title: "Comparing Quarkus to Spring Through the Eyes of JHipster"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2021/01/quarkus-main.png
+---
+# Approach
+According to the website, [Quarkus](https://quarkus.io/) is “A Kubernetes Native Java stack tailored for OpenJDK HotSpot and GraalVM, crafted from the best of breed Java libraries and standards.” I wanted to explore the differences between a typical Spring Boot/Data application and a Quarkus application from a code perspective. (as opposed to performance) 
+
+
+# Generate an Application
+Since JHipster generates an application that is more real than Hello World, I fired up the Quarkus JHipster blueprint. I would like to thank this [post](https://medium.com/quarkify/generate-quarkus-project-with-jhipster-bff4edecb815) for providing step-by-step instructions on how to install the Quarkus blueprint and generate an application.
```

Comment:
```suggestion
Since JHipster generates an actual runnable application out of the box, I fired up the Quarkus JHipster blueprint. I would like to thank this [post](https://medium.com/quarkify/generate-quarkus-project-with-jhipster-bff4edecb815) for providing step-by-step instructions on how to install the Quarkus blueprint and generate an application.
```